plugins/woocommerce-admin/client/wp-admin-scripts/woo-subscriptions-notice/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+
+window.jQuery( document ).ready( function () {
```

Comment:
@hesahesa any reason not to use `domReady` like in https://github.com/woocommerce/woocommerce/blob/09670be305f64460d32bd23b91e3aba6ef8cb163/plugins/woocommerce-admin/client/wp-admin-scripts/woo-plugin-update-connect-notice/index.js#L7-L20

?

Reply:
it's more of a preference (using jquery) like what we did in https://github.com/woocommerce/woocommerce/blob/5bf6f6abfa5a2f3f0abe64740a8ed41ea5af5b3f/plugins/woocommerce-admin/client/wp-admin-scripts/woo-connect-notice/index.js

Reply:
@hesahesa Please do not use jQuery in WooCommerce anymore. We should always prefer plain JS where possible as the plan is to eventually remove jQuery but it cannot be done when we introduce this code.

jQuery also doesn't add anything here you could not have done with plain JS